ft2build.h No such file or directory- make failed

Hello,

I'm trying to build mxe just like shown in the tutorial on my OS: ubuntu 16.04.1 (I just installed the OS).
 Things I did so far:
```
#git clone https://github.com/mxe/mxe.git     //did it in pwd /opt

//Got the requirements
#apt-get install \
    autoconf automake autopoint bash bison bzip2 flex gettext\
    git g++ gperf intltool libffi-dev libgdk-pixbuf2.0-dev \
    libtool libltdl-dev libssl-dev libxml-parser-perl make \
    openssl p7zip-full patch perl pkg-config python ruby scons \
    sed unzip wget xz-utils

#apt-get install g++-multilib libc6-dev-i386

#apt-get install libtool-bin

//build 
#make MXE_TARGETS='x86_64-w64-mingw32.static i686-w64-mingw32.static'

Problem next morning:
Make didn't finished the job. 

Error message on screen:
...
[build]     flann                  x86_64-w64-mingw32.static
[done]      flann                  x86_64-w64-mingw32.static                              25664 KiB      2m50.685s
[build]     flann                  i686-w64-mingw32.static
[done]      flann                  i686-w64-mingw32.static                                17684 KiB      2m28.013s
[download]  fltk
[build]     fltk                   x86_64-w64-mingw32.static
[done]      fltk                   x86_64-w64-mingw32.static                              30996 KiB      0m56.573s
[build]     fltk                   i686-w64-mingw32.static
[done]      fltk                   i686-w64-mingw32.static                                28904 KiB      0m43.098s
[download]  ftgl
[build]     ftgl                   x86_64-w64-mingw32.static

Failed to build package ftgl for target x86_64-w64-mingw32.static!
------------------------------------------------------------
Makefile:659: recipe for target 'libftgl_la-FTFace.lo' failed
make[2]: *** [libftgl_la-FTFace.lo] Error 1
make[2]: Leaving directory '/opt/mxe/tmp-ftgl-x86_64-w64-mingw32.static/ftgl-2.1.3~rc5/src'
Makefile:654: recipe for target 'build-only-ftgl_x86_64-w64-mingw32.static' failed
make[1]: *** [build-only-ftgl_x86_64-w64-mingw32.static] Error 2
make[1]: Leaving directory '/opt/mxe'
real	0m22.896s
user	0m16.608s
sys	0m1.556s
------------------------------------------------------------
[log]      /opt/mxe/log/ftgl_x86_64-w64-mingw32.static
```

Log is attached:

[ftgl_x86_64-w64-mingw32.pdf](https://github.com/mxe/mxe/files/1081032/ftgl_x86_64-w64-mingw32.pdf)

Please tell me what my problem is and how I can fix it.

Sorry to disappoint, but...

```
[narc@gorgon ~/src/mxe]% make sdl2_ttf MXE_TARGETS=x86_64-w64-mingw32.static
[build]     sdl2_ttf               x86_64-w64-mingw32.static

Failed to build package sdl2_ttf for target x86_64-w64-mingw32.static!
------------------------------------------------------------
make[2]: *** [SDL_ttf.lo] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/home/narc/src/mxe/tmp-sdl2_ttf-x86_64-w64-mingw32.static/SDL2_ttf-2.0.14'
Makefile:654: recipe for target 'build-only-sdl2_ttf_x86_64-w64-mingw32.static' failed
make[1]: *** [build-only-sdl2_ttf_x86_64-w64-mingw32.static] Error 2
make[1]: Leaving directory '/home/narc/src/mxe'
real    0m35.381s
user    0m6.472s
sys     0m0.356s
------------------------------------------------------------
[log]      /home/narc/src/mxe/log/sdl2_ttf_x86_64-w64-mingw32.static

Makefile:654: recipe for target '/home/narc/src/mxe/usr/x86_64-w64-mingw32.static/installed/sdl2_ttf' failed
make: *** [/home/narc/src/mxe/usr/x86_64-w64-mingw32.static/installed/sdl2_ttf] Error 1
zsh: exit 2     make sdl2_ttf MXE_TARGETS=x86_64-w64-mingw32.static
[narc@gorgon ~/src/mxe]% uname -a                                                                              
Linux gorgon 4.4.0-31-generic #50-Ubuntu SMP Wed Jul 13 00:07:12 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
[narc@gorgon ~/src/mxe]% git show-ref HEAD                                     
07002736f0c8ad611d727aa9b445215d6e121a10 refs/remotes/origin/HEAD
[narc@gorgon ~/src/mxe]% pastebinit log/sdl2_ttf_x86_64-w64-mingw32.static     
http://paste.ubuntu.com/24897698/
```

Clickable: http://paste.ubuntu.com/24897698/

I *may* have made a small modification to /home/narc/src/mxe/usr/x86_64-w64-mingw32.static/bin/freetype-config to make it `#!/bin/sh -x` (only noticed after pastebinning the log):
```sh
[narc@gorgon ~/src/mxe]% head -n 5 usr/x86_64-w64-mingw32.static/bin/freetype-config 
#! /bin/sh -x
#
# Copyright 2000-2017 by
# David Turner, Robert Wilhelm, and Werner Lemberg.
#
```

--- a/src/freetype-1-fixes.patch
+++ b/src/freetype-1-fixes.patch
@@ -1,0 +1,71 @@
+This file is part of MXE. See LICENSE.md for licensing information.
+
+Contains ad hoc patches for cross building.
+
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Tony Theodore <tonyt@logyst.com>
+Date: Tue, 20 Jun 2017 19:34:02 +1000
+Subject: [PATCH] fix PKG_CONFIG program in freetype-config for cross-compiling
+
+Taken from:
+https://savannah.nongnu.org/bugs/index.php?51274
+
+diff --git a/builds/unix/freetype-config.in b/builds/unix/freetype-config.in
+index 1111111..2222222 100644
+--- a/builds/unix/freetype-config.in
++++ b/builds/unix/freetype-config.in
+@@ -14,7 +14,7 @@ export LC_ALL
+ 
+ 
+ # if `pkg-config' is available, use values from `freetype2.pc'
+-pkg-config --version >/dev/null 2>&1
++%PKG_CONFIG% --version >/dev/null 2>&1
+ if test $? -eq 0 ; then
+   # note that option `--variable' is not affected by the
+   # PKG_CONFIG_SYSROOT_DIR environment variable
+@@ -23,17 +23,17 @@ if test $? -eq 0 ; then
+     export PKG_CONFIG_SYSROOT_DIR
+   fi
+ 
+-  prefix=`pkg-config --variable prefix freetype2`
+-  exec_prefix=`pkg-config --variable exec_prefix freetype2`
++  prefix=`%PKG_CONFIG% --variable prefix freetype2`
++  exec_prefix=`%PKG_CONFIG% --variable exec_prefix freetype2`
+ 
+-  includedir=`pkg-config --variable includedir freetype2`
+-  libdir=`pkg-config --variable libdir freetype2`
++  includedir=`%PKG_CONFIG% --variable includedir freetype2`
++  libdir=`%PKG_CONFIG% --variable libdir freetype2`
+ 
+-  version=`pkg-config --modversion freetype2`
++  version=`%PKG_CONFIG% --modversion freetype2`
+ 
+-  cflags=`pkg-config --cflags freetype2`
+-  dynamic_libs=`pkg-config --libs freetype2`
+-  static_libs=`pkg-config --static --libs freetype2`
++  cflags=`%PKG_CONFIG% --cflags freetype2`
++  dynamic_libs=`%PKG_CONFIG% --libs freetype2`
++  static_libs=`%PKG_CONFIG% --static --libs freetype2`
+ else
+   prefix="%prefix%"
+   exec_prefix="%exec_prefix%"
+diff --git a/builds/unix/unix-def.in b/builds/unix/unix-def.in
+index 1111111..2222222 100644
+--- a/builds/unix/unix-def.in
++++ b/builds/unix/unix-def.in
+@@ -62,6 +62,7 @@ version_info := @version_info@
+ 
+ # Variables needed for `freetype-config' and `freetype.pc'.
+ #
++PKG_CONFIG         := @PKG_CONFIG@
+ REQUIRES_PRIVATE   := @REQUIRES_PRIVATE@
+ LIBS_PRIVATE       := @LIBS_PRIVATE@
+ LIBSSTATIC_CONFIG  := @LIBSSTATIC_CONFIG@
+@@ -102,6 +103,7 @@ NO_OUTPUT := 2> /dev/null
+ $(OBJ_BUILD)/freetype-config: $(TOP_DIR)/builds/unix/freetype-config.in
+ 	rm -f $@ $@.tmp
+ 	sed -e 's|%LIBSSTATIC_CONFIG%|$(LIBSSTATIC_CONFIG)|'   \
++	    -e 's|%PKG_CONFIG%|$(PKG_CONFIG)|'                 \
+ 	    -e 's|%build_libtool_libs%|$(build_libtool_libs)|' \
+ 	    -e 's|%exec_prefix%|$(exec_prefix)|'               \
+ 	    -e 's|%ft_version%|$(ft_version)|'                 \

